extras/beast/unit_test/dstream.hpp

```diff
@@ -130,7 +130,7 @@ class basic_dstream
 {
 public:
     basic_dstream()
-        : std::basic_ostream<CharT, Traits>(&member)
+        : std::basic_ostream<CharT, Traits>(&this->member)
```

Comment:
This does do the right thing - `member` is from `boost::base_from_member` which has already been initialized at this point.